climpred/tests/test_alignment.py

```diff
@@ -0,0 +1,76 @@
+import logging
+
+import xskillscore as xs
+
+from climpred.prediction import compute_hindcast
+
+
+def test_same_inits_initializations(
+    hind_ds_initialized_1d_cftime, reconstruction_ds_1d_cftime, caplog
+):
+    """Tests that inits are identical at all leads for `same_inits` alignment."""
+    with caplog.at_level(logging.INFO):
+        compute_hindcast(
+            hind_ds_initialized_1d_cftime,
+            reconstruction_ds_1d_cftime,
+            alignment='same_inits',
+        )
+        for record in caplog.record_tuples:
+            print(record)
+            # Hard-coded for now, since we know what the inits should be for the demo
+            # data.
+            assert 'inits=1954-01-01 00:00:00-2007-01-01 00:00:00' in record[2]
+
+
+def test_same_inits_verification_dates(
+    hind_ds_initialized_1d_cftime, reconstruction_ds_1d_cftime, caplog
+):
+    """Tests that appropriate verifs are being used at each lead for `same_inits`
+    alignment."""
+    with caplog.at_level(logging.INFO):
+        # Hard-coded for now, since we know what the inits should be for the demo data.
+        FIRST_INIT, LAST_INIT = 1954, 2007
+        compute_hindcast(
+            hind_ds_initialized_1d_cftime,
+            reconstruction_ds_1d_cftime,
+            alignment='same_inits',
+        )
+        for i, record in zip(
+            hind_ds_initialized_1d_cftime['lead'].values, caplog.record_tuples
+        ):
+            assert (
+                f'verif={FIRST_INIT+i}-01-01 00:00:00-{LAST_INIT+i}-01-01 00:00:00'
+                in record[2]
+            )
+
+
+def test_same_inits_disjoint_verif_time(small_initialized_da, small_verif_da):
```

Comment:
With this in mind, it might be helpful to rewrite the logger to show *all* inits and verifs used. But that might be very verbose. Right now it says init=1954-2000 let's say. But it could remove inits within that range if there are missing observations. Or more realistically, in a PM case like yours it's not continuous inits throughout that period. What are your thoughts @aaronspring?

Reply:
you could use logging.DEBUG for that. right now it just shows first and last. good point about missing values. I would say we only need to print the inits once, because the metric anyways fails if `time` differs in `f` and `o`

Reply:
I guess I mean if a user wants to see every init specifically. But I see what you mean maybe. That we use a different level for that so they can select it if they want to see more details. Maybe this is for the future if someone is really interested in seeing that they can ask.